common/classes/com/arjuna/common/internal/util/ClassloadingUtility.java

```diff
@@ -141,8 +141,17 @@ public static <T> T loadAndInstantiateClass(Class<T> iface, String className, St
             }
             commonLogger.logger.warn(message);
         }
+		catch (NoSuchMethodException e) {
+            String message = commonLogger.i18NLogger.warn_common_ClassloadingUtility_5(className, e);
+            if (failOnError) {
+                if (e.getCause() != null) {
+                    throw new Error(message, e.getCause().fillInStackTrace());
+                }
+            }
+            commonLogger.logger.warn(message);
+        }
 
-        return instance;
+		return instance;
```

Comment:
Please let me know if it would be helpful for me to identify all the places where the tabs should be changed to spaces? Otherwise, please can I ask you to take a look through the diff updating those parts that are new and use tabs to use spaces instead (4 spaces per tab)?

Thank you very much for your pull request!